src/js/components/common/configurationobject.js

```diff
@@ -16,26 +16,32 @@ const useStyles = makeStyles()(theme => ({
   }
 }));
 
-const ValueColumn = ({ value, setSnackbar }) => {
+const cutoffLength = 100;
+const ValueColumn = ({ value = '', setSnackbar }) => {
   const [tooltipVisible, setTooltipVisible] = useState(false);
+  const isComponent = React.isValidElement(value);
   const onClick = () => {
     if (setSnackbar) {
       let copyable = value;
-      if (React.isValidElement(value)) {
+      if (isComponent) {
         copyable = value.props.value;
       }
       copy(copyable);
       setSnackbar('Value copied to clipboard');
     }
   };
+  let shownValue = value;
+  if (!isComponent) {
+    shownValue = <div title={value}>{value.length > cutoffLength ? `${value.substring(0, cutoffLength - 3)}...` : value}</div>;
```

Comment:
Q: Why the post `...` ? :thinking:

Reply:
it's commonly treated as a sign for "more content" if it was left to CSS to handle this via overflow that would be the default "ellipsis" so `...`